examples/iDynTreeWrappers/visualizeRobot.m

```diff
@@ -0,0 +1,93 @@
+% This script is an example on how to use the matlab iDynTree visualization
+
+% insert the path where the meshes may be found, it repaces what is require
+% d to complete the path inside the urdf visual mesh field.
+meshFilePrefix=[getenv('ROBOTOLOGY_SUPERBUILD_SOURCE_DIR') '/build-linux/install/share'];
+
+% create KinDynModel variable from another script
+icubModelPath=[ getenv('ROBOTOLOGY_SUPERBUILD_SOURCE_DIR') '/build-linux/install/share/iCub/robots/iCubGazeboV2_5/'];
+fileName='model.urdf';
```

Comment:
I  think it is more clear if you do something like that: 

~~~
% This example requires to use the URDF models installed by https://github.com/robotology/icub-models
% To do so, please substitute in the following the location of the install prefix of icub-models:
icubModelsInstallPrefix = "";
meshFilePrefix = [icubModelsInstallPrefix '/share']
icubModelPath = [icubModelsInstallPrefix '/share/iCub/robots//iCubGazeboV2_5/']
~~~

Reply:
This would require the user to have iCubModels installed for the test no? I'm planning to add a urdf with the stl files inside the example folder so that it is self contained and runs out of the box. Is that fine for you?

Reply:
I also added the suggestion you put here